utils/src/main/scala/za/co/absa/enceladus/utils/testUtils/SparkTestBase.scala

```diff
@@ -69,12 +71,19 @@ trait SparkTestBase { self =>
   Logger.getLogger("org").setLevel(Level.WARN)
   Logger.getLogger("akka").setLevel(Level.WARN)
 
+  protected val showIsActive = false
```

Comment:
Please, add a comment explaining the purpose of `showIsActive` and `showDataFrame`.
I'm guessing the usage is changing it to `true` every time we want to see the debug output?

Why not just use `logger.debug(...)`? You can redirect `df.show(...)` to a string and then use the debug logger. The schema can also be exported to a string with `df.schema.treeString`

Reply:
```scala
  /** Same as df.show(), bu the output goes to a string */
  def showString(df: DataFrame, numRows: Int = 20): String = {
    val outCapture = new ByteArrayOutputStream
    Console.withOut(outCapture) {
      df.show(numRows, truncate = false)
    }
    new String(outCapture.toByteArray).replace("\r\n", "\n")
  }
```